testEnvironment.js

```diff
@@ -0,0 +1,5 @@
+require('dd-trace').init({
+  startupLogs: false,
+})
+
+module.exports = require('jest-environment-node')
```

Comment:
Don't we need this in the dependencies?

Reply:
sry for the delay in the answer: yeah that's a good question. Jest does not refer to adding the dependency (https://jestjs.io/docs/en/configuration#testenvironment-string) plus the dependency seems to be hoisted when using `jest`. WDYT?

Reply:
I'd think better be explicit with what we use.
If it we can add it to the dependencies, we should.

Reply:
yeah that makes sense. `jest` uses `jest-environment-node` or `jest-environment-jsdom` under the hood if you don't specify anything though, so it is being used anyway. But I'm all for being explicit 👍

Reply:
> jest uses jest-environment-node or jest-environment-jsdom under the hood

Indeed, and by being explicit ourselves, if ever jest decide not to use it anymore, we won't have an error, because it wouldn't be imported anymore.